code/game/machinery/washing_machine.dm

```diff
@@ -79,9 +79,8 @@
 	if(istype(W,/obj/item/weapon/pen/crayon) || istype(W,/obj/item/weapon/stamp))
 		if( state in list(	1, 3, 6 ) )
 			if(!crayon)
-				user.drop_item()
+				user.drop_from_inventory(W,src)
 				crayon = W

```

Comment:
Erm, this isn't correct. This should be `crayon.forceMove(src)`

Reply:
user.drop_from_inventory(item,target) acts as a item.forceMove(target)